internal/connect/product.go

```diff
@@ -5,11 +5,17 @@ import (
 	"strings"
 )
 
-// Product represents an installed product or product information from API
-type Product struct {
+// BasicProduct represents basic information on product, mostly for API requests
```

Comment:
maybe this comment could also mention that BasicProduct is used instead of Product to avoid sending unneeded bool json or something